Doc/library/asyncio-future.rst

```diff
@@ -65,7 +65,7 @@ Future Functions
       and *loop* is not specified and there is no running event loop.
 
 
-.. function:: wrap_future(future, *, loop=None)
+.. function:: wrap_future(future, /, *, loop=None)
```

Comment:
Why this change?

Reply:
Conceptually it's the right change and that's how I should have documented this function when I wrote these docs. But I agree this can go in a separate PR, was a knee-jerk edit.